README.md

```diff
@@ -81,12 +97,12 @@ Through some mechanism in your app, you determine what two users should be stree
 To make a call to user "tom", do the following:
 
 ```swift
-   let state = StreemStateBuilder()
-        .with(myRole: .LOCAL_CUSTOMER)
-        .with(remoteUserId: "tom", andRole: .REMOTE_PRO)
-        .build()
-
-    Streem.openStreem(state)
+    let user = StreemUser(name: "Tom Smith", id: "tom")
```

Comment:
This initializer needs to be written.

Currently our demo app gets an already filled-in `StreemUser` object from `getRecentlyIdentifiedUsers()`.

How much of the `User` struct do we need to fill in before starting a streem? Is `userId` sufficient?

Reply:
I think we can stick with just `userId` right now, but understand that it should be their `userId` (our `externalUserId`).  If this isn't the way they code is written right now, we should make it such.  I'd like as much of our public API to use their user ids as possible.